plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1238,7 +1238,7 @@ public String getMemBalloonStatsPeriod() {
         @Override
         public String toString() {
             StringBuilder memBalloonBuilder = new StringBuilder();
-            memBalloonBuilder.append("<memballoon model='" + memBalloonModel + "' autodeflate='on'>\n");
+            memBalloonBuilder.append("<memballoon model='" + memBalloonModel + "'>\n");
```

Comment:
@rohityadavcloud does this mean, the blocker issue https://github.com/apache/cloudstack/issues/7794 is fixed here?

Reply:
Possibly - we need to check/test cc @DaanHoogland

Reply:
@harikrishna-patnala , yes as shown by 
https://github.com/apache/cloudstack/pull/7810#issuecomment-1664869520